data/remnant/remnant 1 introduction.txt

```diff
@@ -1302,19 +1302,23 @@ mission "Remnant: Deep Surveillance"
-			`	You ascend the stairs of the Institute, the massive building looming over you as you approach. Its more ancient design makes the building stand out from the rest of the district, the entrance being guarded by massive marble pillars that manage to not clash with the metallic sheen of the rest of the buildings. The interior is similarly impressive, with a massive lobby area centered by an impressive water fountain.`
-			`	You approach one of the receptionist desks and ask to see Ivan Haataja. "It looks like he's available right now," she says. "We'll let him know you're coming and have someone here to escort you to Doctor Haataja's office shortly."`
-			`	You're escorted through what seems like endless hallways and stairwells, the entire building being as impressively decorated as the lobby, before finally coming to Ivan's office. You thank the escort for bringing you here (as you would have been lost for hours without him, surely why they have people just for escorting guests) and knock on Ivan's door.`
-			`	Ivan opens the door with a big smile on his face. "Captain <last>! So great to see you here. Please, have a seat." Ivan ushers you inside his office, an impressively large room with a window wall overseeing a beautiful courtyard. You sit down in a cushioned chair on the opposite side of a desk. You lean back into the chair and get comfortable. "I know the chairs are cozy," Ivan says. "Don't fall asleep on me though."`
+			choice
+				`	"Yes, please."`
+			`	You get directions from the receptionist and thank him for his time. It turns out that this Institute is in the heart of the university district of the spaceport city. Several planets in the Deep, Paradise Worlds, and even many near Earth have a similar university district, but the one on Valhalla is the largest of its kind in the entire Republic.`
+			`	As you enter the district, you note that the architecture here is much more appealing than the brutalistic style of Ivan's old office building. Each edifice consists of more curves than right angles, with a textured metallic sheen decorating its surface. Brilliant facades scatter sunlight to give the area a warm glow, as opposed to directing it right into your eyes if you happen to be standing in the wrong spot. Even among all the metallic structures, there is still a good amount of greenery: well-trimmed bushes and trees line the streets, and many of the buildings are topped with gardens.`
+			`	You ascend the stairs of the Institute, the massive building looming over you as you approach. Its more ancient design makes the building stand out from the rest of the district; the entrance is guarded by massive marble pillars that manage to not clash with the metallic sheen of the rest of the structures nearby. The interior is similarly lavish, with a sprawling lobby area peppered with reception areas and centered by a massive fountain.`
```

Comment:
First sentence doesn't feel great - "as you approach" but you just said that you were ascending...